packages/core/src/client.ts

```diff
@@ -45,7 +46,7 @@ import {
 } from './utils';
 
 import { DocumentNode } from 'graphql';
-import { Target } from './utils/Target';
+import { subscriptionT } from 'wonka/dist/types/src/Wonka_types.gen';
```

Comment:
This feels like an incorrect import to me

Reply:
Good catch